openapi.yaml

```diff
@@ -644,6 +660,7 @@ paths:
   #              $ref: "#/components/schemas/Error"
 components:
   schemas:
+    # Bus route schema
```

Comment:
these comments are redundant and don't add info

Reply:
For me, they help me visually break up the file and help with readabilty. If this is just a me thing though I can remove it

Reply:
personally I agree with evan, the comment is just one more thing to read and only adds information already conveyed by the code. I wouldn't however be against breaking up the file by adding a newline between each one, like you've done, because that does improve readability for me.

Reply:
2-1, I'll remove them